doc/user/content/third-party/metabase.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Using Metabase"
+description: "Get details about using Materialize with Metabase"
+menu:
+  main:
+    parent: 'third-party'
+---
+
+You can use [Metabase] to create business intelligence dashboards using the
+real-time data streams in your Materialize instance. To get started, check out
+our [metabase-materialize-driver].
+
+## Limitations
+
+Our [metabase-materialize-driver] uses a forked version of pgjdbc to work. This
+should not affect users trying to use the driver, only those who wish to make updates.
+Track our progress in un-forking the pgjdbc code [here](https://github.com/MaterializeInc/materialize/issues/3727).
```

Comment:
Personally I think we can avoid mentioning this, since it's probably not relevant to 99% of people reading the docs!

Reply:
@benesch I think the idea here is to express why we don't offer production-level support for Metabase; I think that's worthwhile and a good practice for any/all tools without our highest tier of support.